### PR TITLE
Add metaport bridge docs

### PR DESCRIPTION
**Pull request by cstrangedk**

### CODE REVIEW

File: .vscode/settings.json
Comment (TheGreatAxios) [inline .vscode/settings.json:1]: Did you mean to commit this file?

File: components/develop/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -37,6 +37,7 @@
 ** xref:ml.adoc[Using SKALE-ML]
 ** xref:ima::index.adoc[Performing IMA Bridge transfers]
 ** xref:metaport::index.adoc[Using Metaport Widget]
+** xref:using-metaport-bridge.adoc[Using Metaport Bridge]
```
Comment (TheGreatAxios) [inline components/develop/modules/ROOT/nav.adoc:40]: Suggestion Only, Not mandatory change - Flip these so that Bridge comes before Widget

**Review by TheGreatAxios — APPROVED** [review]
Approved with suggestions. 

.vscode file - Have not seen this before in this repo, should it be added to .gitignore before merge?

File: components/develop/modules/ROOT/pages/using-metaport-bridge.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
+= Using Metaport Bridge
+
+== What is the Metaport Bridge
+
+The  Metaport Bridge is a tool that simplifies the process for end-users to transfer tokens between Ethereum and any SKALE Chain or Dapp, as well as facilitating token transfers between different SKALE Chains.
```
Comment (SKALE-jace) [inline components/develop/modules/ROOT/pages/using-metaport-bridge.adoc:5]: extra space to remove

File: components/develop/modules/ROOT/pages/using-metaport-bridge.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+[NOTE]
+Note that fallback-url and fallback-text should be encoded with encodeURIComponent().
+
+== Examples
+
+For transferring from Calypso to Europa:
+`https://bridge.skale.network/#/bridge/transfer/honorable-steel-rasalhague/elated-tan-skat` 
+
+
+For transferring from Ethereum to NFT Trade:
```
Comment (SKALE-jace) [inline components/develop/modules/ROOT/pages/using-metaport-bridge.adoc:40]: Would suggest `.. to NFTrade on Calypso.` So it's an demonstrating that a chain can be referenced this way implicitly, but in the end still ends up on the same chain.

File: components/develop/modules/ROOT/pages/using-metaport-bridge.adoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+Developers can use the Bridge as an alternative to implementing their own Metaport widget.
+
+== How is it different from the Metaport Widget?
+
+The Metaport widget is an embeddable React component that can be used by SKALE Community developers to guide users through a SKALE IMA bridge transaction, without having to leave the dapp.
+
+The Metaport Bridge implements the Metaport widget - just as other dapps building on SKALE can also implement the Metaport widget into their user flows. The Bridge is more like a general tool, allowing users to navigate any Bridge transfer within the SKALE Network. The Metaport Bridge provides zero-cost, near-instant transfers of assets and data between connected SKALE chains, as well as fast-finality transfers between SKALE Chains connected to Ethereum mainnet (ETH gas fees apply to Ethereum mainnet transactions).
+
+== Sending users with fall back
```
Comment (SKALE-jace) [inline components/develop/modules/ROOT/pages/using-metaport-bridge.adoc:15]: fallback

**Review by SKALE-jace — CHANGES_REQUESTED** [review]
made a few comments on spelling/spacing corrections and wording suggestion, but otherwise this looks good

**Review by SKALE-jace — COMMENTED** [review]
already merged, but looks good!

**Pull request merged**